index.js

```diff
@@ -159,28 +166,34 @@ function makeRequest() {
   var additionalParams = JSON.parse(argv.additionalParams);
   var body = JSON.parse(argv.body);
 
+  if (argv.accessTokenHeader) {
+    const tokenHeader = {};
+    tokenHeader[argv.accessTokenHeader] = userTokens.accessToken;
+    additionalParams.headers = Object.assign({}, additionalParams.headers, tokenHeader);
+  }
+
   apigClient
     .invokeApi(params, argv.pathTemplate, argv.method, additionalParams, body)
     .then(function(result) {
       console.dir({
         status: result.status,
         statusText: result.statusText,
-        data: result.data
+        data: JSON.stringify(result.data)
```

Comment:
I was leaving it as an object since Node does a good job of formatting objects in the console. Any reason why you wanted to `stringify` it?

Reply:
Hey it seems that when I don't stringify I just get the top level properties of the result displayed and I wanted to see the full (deep) object. I'm hitting a graphql endpoint so the response looks like this:

```
Authenticating with User Pool
Getting temporary credentials
Making API request
{ status: 200,
  statusText: 'OK',
  data:
   { data: { vendors: [Array] },
     extensions: { tracing: [Object] } } }
```

The interesting part of the result is in that Array. I can switch this back to not use stringify but maybe it should be another option.